### DISCUSSION TITLE
Keys pressed from the keyboard when the event occurs

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [lit/lit#3292] [Ideas]
# List of Keys pressed in parameters

```ts
html `<button @click=${func}><\button>`

function func(event) {
  console.log(event.keys); // ['ctrl', 'shift']
}
```

**Comment by drk-mtr** [comment]
Is this a question? Is the question "how do I capture keypress events"?

The event you're passing here is a MouseEvent, so it won't contain keypress info. I believe you need to subscribe to `keyup` events, maybe in the `connectedCallback()` lifecycle method? Or use `@keyup` on the element?

**Reply by justinfagnani** [reply]
MouseEvent does have properties representing whether the modifier keys were pressed: `altKey`, `ctrlKey`, and `metaKey`: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/altKey

**Reply by drk-mtr** [reply]
Thanks for correcting me, I should really add an "I'm a beginner but..." disclaimer to all posts ha.

**Reply by MM25Zamanian** [reply]
thanks, I'm a beginner but, I'm not use `React` 😂.

Just this [feature](https://vuejs.org/guide/essentials/event-handling.html#key-modifiers) is interesting for me
